ts-frontend/src/components/Map.tsx

```diff
@@ -164,6 +196,28 @@ export const Map = () => {
 
   const { qLocations, setQLocations } = useQLocations();
 
+  const fetchCurrentUserDetails = () => {
+    const userDetails = localStorage.getItem("user");
+    if (userDetails) {
+      fetch(`${BASE_API_URI}/api/user/${JSON.parse(userDetails)._id}`, {
+        headers: {
+          "Content-Type": "application/json",
+          Authorization: `Bearer ${localStorage.getItem("jwt")}`,
+        },
+      })
+        .then((res) => res.json())
+        .then((data) => {
+          if (data.error) {
+            console.log(data.error);
+          } else {
+            setcurrUser(data.user);
+            localStorage.setItem("user", JSON.stringify(data.user));
+          }
+        })
+        .catch((err) => console.log(err));
+    }
+  };
+
   const updateDetails = () => {
```

Comment:
this function is used to get the details of the the users that are queried  by the current client with the values taken from the distance, age slider and college, gender drop-down.

we also have a ` updateCurrentLocation` function that updates the currentUser (user with id: _currentUserUUID_) location and writes it to the Postgres through the ts-backend server.

these two function names are ambiguous so,
rename this  `updateDetails` function to `updateDetailsOfQueriedUsers` for better clarity

Reply:
done